### DISCUSSION TITLE
I used clipboard when used ToolTipOpt.ahk then I got a bug says setclipboarddataerror, just simply adding a line of clipboard changing code in your exemple.

### DISCUSSION THREAD

**Discussion by hengxianqin11** [community/community#128740] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

here is the tool:
https://github.com/Ixiko/AHK-libs-and-classes-collection/blob/master/libs/o-z/ToolTipOpt.ahk
here is your exemple:
https://www.autohotkey.com/boards/viewtopic.php?t=4777

here is my code,just adding three lines "clipboard := '1/2/3' ":
clipboard := "1" 
ToolTip Standard tooltip
Sleep 1000

clipboard := "2" 
ToolTipFont("s20", "Comic Sans MS")
ToolTip ToolTip with custom font
Sleep 1000

clipboard := "3" 
ToolTipColor("Black", "White")
ToolTip ToolTip with custom font and color
Sleep 1000

**Comment by ebndev** [comment]
Hi @hengxianqin11, thanks for being a part of the GitHub Community!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from specific project repo, is that right?

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

If you don't mind, would you mind letting us know how you ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your Discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board. Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
